src/test/java/htsjdk/cram/io/ExternalCompressionTest.java

```diff
@@ -1,29 +0,0 @@
-package htsjdk.cram.io;
```

Comment:
moved from `htsjdk.cram.io` to `htsjdk.samtools.cram.io` like the class it's testing